index.html

```diff
@@ -12,6 +12,9 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```

Comment:
please use https://www.npmjs.com/package/vite-plugin-fonts for setting up google fonts.

Reply:
opened this up as a new issue as I saw it was from the previous PR: https://github.com/open-sauced/hot/issues/225